package/components/editor-utils.tsx

```diff
@@ -758,6 +755,15 @@ export const TextColor = ({
           />
         </div>
       ))}
+      <div
+        onClick={() => {
+          editor.chain().focus().unsetColor().run();
+          setVisibility(IEditorTool.NONE);
+        }}
+        className="flex w-full items-center gap-1 cursor-pointer mt-2 mb-1 text-sm text-gray-800 hover:bg-gray-100 rounded-md p-1"
+      >
+        <Ban size={18}></Ban> None
+      </div>
```

Comment:
why we added this ?

Reply:
To unset for Text Color as similar to Highlighter